docs/bucket/notifications/README.md

```diff
@@ -424,8 +425,89 @@ key         |                      value
 (1 row)
 ```
 
+<a name="MySQL"></a>
+## Publish Minio events via MySQL
+
+Install MySQL from [here](https://dev.mysql.com/downloads/mysql/). For the purpose of illustrating sending events from Minio to MySQL, we set the root password as `password` and created a database called `miniodb` to store the events.
+
+### Step 1: Add MySQL server endpoint configuration to Minio
+
+The default location of Minio server configuration file is ``~/.minio/config.json``. Under the `notify` key, in the `mysql` sub-key, we may add as many MySQL server endpoint configurations as desired by providing an identifier for the server and an object of per-server configuration parameters.
```

Comment:
How about splitting this information as -- 

The default location of Minio server configuration file is ``~/.minio/config.json``. Find the mysql sub-key under the notify key in the config file and update the relevant sections.